src/test/scala/nl.knaw.dans.easy.validatebag/rules/structural/StructuralRulesSpec.scala

```diff
@@ -74,7 +74,19 @@ class StructuralRulesSpec extends TestSupportFixture {
 
   it should "succeed exactly the files/directories specified are present" in {
     testRuleSuccess(
-      rule = containsNothingElseThan(Paths.get("metadata"), Seq("dataset.xml", "files.xml")),
+      rule = containsNothingElseThan(Paths.get("metadata"),
+        Seq(
+        "dataset.xml",
+        "files.xml",
+        "amd.xml",
+        "emd.xml",
+        "license.txt",
```

Comment:
You don't expect pdf versions?

Reply:
i was wondering that. if there are additional licenses, they will need to have a postfix or similar to make the names unique. i thought to add that when we encounter the case ...